addon/storages/memory.js

```diff
@@ -23,7 +20,9 @@ export default ObjectProxy.extend(StorageMixin, {
 	 * @type {Object}
 	 * @default Ember.Object
 	 */
-	initialContent: computed(() => Ember.Object.create()),
+	// "ember/no-arrow-function-computed-properties" conflicts with "prefer-arrow-callback"
+	// eslint-disable-next-line ember/no-arrow-function-computed-properties
+	initialContent: computed(() => EmberObject.create()),
```

Comment:
```suggestion
	init() {
	  this._super(...arguments);
	  this.initialContent = EmberObject.create();
	}
```

Reply:
I couldn't manage to make this one work, seems that no matter where I put the `_super`, there is no `initialContent` at some point during the tests and they break cathastropically. I'd say lets this one stay for now.